test/integration_testcase.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package test
+
+import (
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+var (
+	UpdateGolden *bool = flag.Bool("update", false, "update golden files")
+)
+
+// To manage a test case directory structure and content
```

Comment:
Use the recommended comment format from `go lint` here:  `IntegrationTestCase ...`